includes/process_config.inc.php

```diff
@@ -1,61 +0,0 @@
-<?php
```

Comment:
How come all of this file has been removed?

Reply:
Its replacement is inside Config.php and is `process_config.inc.php`never called right now.